README.md

```diff
@@ -1,128 +1,60 @@
-- [recoveryservices](/arm/recoveryservices)
-- [redis](/arm/redis)
-- [resources/features](/arm/resources/features)
-- [resources/links](/arm/resources/links)
-- [resources/locks](/arm/resources/locks)
-- [resources/policy](/arm/resources/policy)
-- [resources/resources](/arm/resources/resources)
-- [resources/subscriptions](/arm/resources/subscriptions)
-- [scheduler](/arm/scheduler)
-- [search](/arm/search)
-- [servermanagement](/arm/servermanagement)
-- [service-map](/arm/service-map)
-- [servicebus](/arm/servicebus)
-- [sql](/arm/sql)
-- [storage](/arm/storage)
-- [storageimportexport](/arm/storageimportexport)
-- [trafficmanager](/arm/trafficmanager)
-- [web](/arm/web)
-
-- [keyvault dataplane](/dataplane/keyvault)
-
-## Azure Service Management (ASM), aka classic deployment
-
-[About ASM](/management/README.md)
-
-- [affinitygroup](/management/affinitygroup)
-- [hostedservice](/management/hostedservice)
-- [location](/management/location)
-- [networksecuritygroup](/management/networksecuritygroup)
-- [osimage](/management/osimage)
-- [sql](/management/sql)
-- [storageservice](/management/storageservice)
-- [virtualmachine](/management/virtualmachine)
-- [virtualmachinedisk](/management/virtualmachinedisk)
-- [virtualmachineimage](/management/virtualmachineimage)
-- [virtualnetwork](/management/virtualnetwork)
-- [vmutils](/management/vmutils)
-
-## Azure Storage SDK for Go
-
-[About Storage](/storage/README.md)
-
-- [storage](/storage)
 
-# Installation
+This is Microsoft Azure's core repository for hosting Go packages which offer a more convenient way of targeting Azure
+REST endpoints. Here, you'll find a mix of code generated by [Autorest](https://github.com/Azure/autorest) and hand
+maintained packages.
 
-- [Install Go 1.7](https://golang.org/dl/).
+> **NOTE:** This repository is under heavy ongoing development and is likely to break over time. We currently do not
+have any releases yet. If you are planning to use the repository, please consider vendoring the packages in your
+project and update them when a stable tag is out.
 
+# Installation
+- If you don't already have it, install [the Go Programming Language](https://golang.org/dl/).
 - Go get the SDK:
 
 ```
-$ go get -d github.com/Azure/azure-sdk-for-go
+$ go get -u github.com/Azure/azure-sdk-for-go
 ```
 
-> **IMPORTANT:** We highly suggest vendoring Azure SDK for Go as a dependency. For vendoring dependencies, Azure SDK for Go uses [glide](https://github.com/Masterminds/glide). If you haven't already, install glide. Navigate to your project directory and install the dependencies.
+> **IMPORTANT:** We highly suggest vendoring Azure SDK for Go as a dependency. For vendoring dependencies, Azure SDK
+for Go uses [glide](https://github.com/Masterminds/glide).
 
-```
-$ cd your/project
-$ glide create
-$ glide install
-```
+# Versioning
+## SDK Versions
+The tags in this repository are based on, but do not conform to [SemVer.org's recommendations](http://semver.org/).
```

Comment:
It's embarrassing, but it is the truth. I'd rather be slightly self-deprecating than slightly misleading. :)